site/source/tutorials/1030_routing/index.md

```diff
@@ -126,10 +137,16 @@ w(Link, { to: 'outlet-name', params: { paramName: 'value' } });
 
 {% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' line:4 %}
 
-{% instruction 'Replace the render function in `App.ts`.' %}
+{% instruction 'Replace the `render` function in `App.ts`.' %}
 
 {% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:37-55,59-64 %}
 
+The updated block of code continues to render the Banner, Worker Form and Worker Container widgets, and additionally renders three `Link` widgets:
```

Comment:
Should Banner etc. be in backticks?

Reply:
In this case I was trying to word it as UI components rather than technical terms!